src/modules/end-user-guide/pages/create-application.adoc

```diff
@@ -0,0 +1,247 @@
+include::ROOT:partial$variables.adoc[]
+
+= Application Management
+
+An application on the {PRODUCT_NAME} low-code platform is a link to a resource's home page. Each application has a corresponding thumbnail for each resource.
+
+[annotation,role="data-zoomable"]
+----
+{
+  "image": {
+    "rel": "application/c-applications.png",
+    "w": 1920,
+    "h": 1080
+  },
+  "view": {},
+  "annotations": []
+}
+----
+
+== Creating an application
+
+To create a new application, follow the steps below.
+
+Access your {PRODUCT_NAME} instance and click the btn:[Admin Area] application.
+
+[annotation,role="data-zoomable"]
+----
+{
+  "image": {
+    "rel": "application/c-application.png",
+    "w": 1920,
+    "h": 1080
+  },
+  "view": {},
+  "annotations": [{
+    "kind": "box",
+    "x": 967,
+    "y": 356,
+    "w": 204,
+    "h": 166
+  }]
+}
+----
+
+Locate and click on the btn:[Applications] button.
+
+[annotation,role="data-zoomable"]
+----
+{
+  "image": {
+    "rel": "application/application-button.png",
+    "w": 1920,
+    "h": 1080
+  },
+  "view": {},
+  "annotations": [{
+    "kind": "box",
+    "x": 13,
+    "y": 179,
+    "w": 161,
+    "h": 24
+  }]
+}
+----
+
+Locate and click on the btn:[new] button. 
+
+[annotation,role="data-zoomable"]
+----
+{
+  "image": {
+    "rel": "application/new-application.png",
+    "w": 1920,
+    "h": 1080
+  },
+  "view": {},
+  "annotations": [{
+    "kind": "box",
+    "x": 1369,
+    "y": 64,
+    "w": 50,
+    "h": 24
+  }]
+}
+----
+
+Complete the form with the application's name and click the btn:[submit] button.
+
+[annotation,role="data-zoomable"]
+----
+{
+  "image": {
+    "rel": "application/create-application.png",
+    "w": 1920,
+    "h": 1080
+  },
+  "view": {},
+  "annotations": [{
+    "kind": "box",
+    "x": 1451,
+    "y": 216,
+    "w": 64,
+    "h": 25
+  }]
+}
+----
+
+== Editing an application.
+
+To edit or make changes to the details of an application, follow the following steps.
+
+Access your {PRODUCT_NAME} instance and click the btn:[Admin Area] application.
```

Comment:
Personally, I'm not the biggest fan of this:

> To edit or make changes to the details of an application, follow the following steps.
> 
> Access your {PRODUCT_NAME} instance and click the btn:[Admin Area] application.

because 

> ..., follow the following steps

is redundant as we already assume we'll be learning about it here.

What I would do is merge the two sentences in the lines of:

`... make changes to the details of an application, Access your {PRODUCT_NAME} instance and click the btn:[Admin Area] application.`